views/books.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Book List</title>
+  <link rel="stylesheet" type="text/css" href="stylesheets/books.css" />
+</head>
+<body>
+  <h1>Reading List</h1>
+  <h3>Currently Reading</h3>
+  <p id="currently-reading">
+    <!-- currently reading will be rendered here -->
+  </p>  
+
+  <h3>To Be Read</h3>
+  <ul id="book-list">
+    <!-- TBR books will be rendered here -->
+  </ul>
+
+  <h3>Completed</h3>
+  <ul id="completed-list">
+    <!-- Already read books will be rendered here -->
+  </ul>
+
+  <script>
```

Comment:
So there is a lot going on below- I think we could use something more sophisticated but I can't be bothered at the moment